pom.xml

```diff
@@ -122,6 +122,11 @@
             <artifactId>file-download-wrapper</artifactId>
             <version>3.0.1</version>
         </dependency>
+        <dependency>
+            <groupId>in.virit</groupId>
```

Comment:
IMO it's not worth it to use an addon for this. A JS call would be sufficient.

Reply:
IMO an addon is easier than JS

Reply:
That is why the addon exists

Reply:
As a user interested in this use case, it's just extra work to go and figure out how to add this addon. I would have to notice the comment in the code, go to the directory, register, add the addon to my pom.xml.

I don't think the addon provides much benefit here.

Reply:
I fully disagree. By using an addon you get something that somebody else thought through. Do you need one of the other methods in the addon? They are already there. Is there a bug? Maybe it was already fixed because you are using code that other people also are using.

Rarely ever is there a point to implement yourself what is already in an addon

Reply:
Let me put it this way: I think that using the addon is suboptimal, but the recipe is LGTM in terms of it fulfilling its purpose. So I'd say let's merge it as it is.